src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -408,23 +408,25 @@ def _crawler(self) -> Iterable[MigratedGroup]:
 
     def validate_group_membership(self) -> list[dict]:
         workspace_groups_in_workspace = self._workspace_groups_in_workspace()
-        account_groups_in_account = self._account_groups_in_account()
+        account_groups_in_account = self._account_groups_in_account("id,displayName,externalId,members")
```

Comment:
Be careful when fetching unbounded attributes like "members" we had timeout problems with the Workspace API in the past because of this. We may run into the same issue there

Reply:
Additionally, I don't think you need the externalId field

Reply:
corrected. I pull the members now with a separate Get. It's much faster and we don't risk a timeout